pixart.js

```diff
@@ -1 +1,23 @@
+var brush = document.querySelector( ".brush" );
 
+// I thought this works for both one and two of the commits?
+// After looking around I could only find a jquery solution...
+var setColorClick = function(){
+    button.addEventListener("click" , function(evt){		    button.addEventListener("click" , function(evt){
+    })
+  };
+
+
+
+// Thank you Mozilla, Doucment.creatElement() helped solve this one.
+      var createDivs = function() {
+   for (i = 0; i < 20; i) {
+     var newDiv = document.createElement("div");
```

Comment:
Your for loop here is incorrect

First, you should really only be declaring i as a local variable. Also you are not incrementing your for loop anywhere! It should read something like this:

`for (var i = 0; i < 20; i++)
`